public/app/plugins/panel/canvas/editor/element/QueryParamsEditor.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FormEvent, useState } from 'react';
```

Comment:
I think we can remove this file (which will also cleanup associated betterer issues)

Reply:
oops, merging conflicts left over!